modules/auxiliary/admin/mssql/mssql_idf.rb

```diff
@@ -22,9 +22,8 @@ def initialize(info = {})
         This module will search the specified MSSQL server for
         'interesting' columns and data.
 
-        The module has been tested against SQL Server 2005 but it should also work on
-        SQL Server 2008. The module will not work against SQL Server 2000 at this time,
-        if you are interested in supporting this platform, please contact the author.
+        This module has been tested against the latest SQL Server 2019 docker container image (22/04/2021).
+        If you are interested in supporting this platform, please contact the author.
```

Comment:
This line doesn't quite make sense now, I believe they were suggesting to contact the author if you wanting to work on adding support for SQL Server 2000

Reply:
Oh ok, my bad, I see that now. 
Reading is hard :upside_down_face: